### DISCUSSION TITLE
我的2022 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#136] [Announcements]
# 我的2022 · BMPI

我的2022年度个人总结

https://www.bmpi.dev/self/annual-summary/2022/

**Comment by bytewish** [comment]
博主，你在文章里画的好看的表格都是怎么弄出来的，感觉非常精致啊

**Reply by madawei2699** [reply]
这篇文章没表格，这些图片是系统自动生成的啊

**Comment by lionoggo** [comment]
同问，对up主的这套系统很感兴趣，是自己开发的量化？

**Comment by BestQwerty** [comment]
优秀呀，博客风格大爱
